frontend/website/src/Nav.re

```diff
@@ -1,13 +1,22 @@
 /* Nav styles adapted from https://medium.com/creative-technology-concepts-code/responsive-mobile-dropdown-navigation-using-css-only-7218e4498a99 */
 
```

Comment:
I think this comment is killing syntax highlighting, does a single line one behave better?